### PR TITLE
String length

### PR DESCRIPTION
**Pull request by chapeupreto**
Get string length.

### CODE REVIEW

File: src/Strings.php [hunk 1]
```diff
@@ -549,6 +549,17 @@ function left($string, $n)
...
 
+/**
+ * Return the length of a string
+ * @param  string $string the input string
+ * @return int         the length of the input string
+ * @author Rod Elias <rod@wgo.com.br>
+ */
+function length($string)
+{
+    return function_exists('mb_strlen') ? mb_strlen($string) : strlen($string);
```
Comment (gcds) [inline src/Strings.php]: Please look at other functions which uses mb_ functions and apply same options to yours. Thanks :+1:

**Comment by chapeupreto** [comment]
Sorry! Thanks for letting me know about that. Please give it a try again.

**Comment by gcds** [comment]
Sorry, but tests fails so I will not merge this until tests passes

**Comment by chapeupreto** [comment]
I see. Tests fixed now.

**Pull request merged**